config/migrations/2023/20230713123426-update-lekp-aanvraag-step-deadline.sparql

```diff
@@ -0,0 +1,24 @@
+PREFIX m8g: <http://data.europa.eu/m8g/>
+
+DELETE {
+  GRAPH ?g {
+    ?s m8g:startTime ?start .
+    ?s m8g:endTime ?end .
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?s m8g:startTime "2023-09-01T21:59:59Z"^^xsd:dateTime .
+    ?s m8g:endTime "2023-12-06T21:59:59Z"^^xsd:dateTime .
```

Comment:
I haven't checked this yet, but I think this should be `"2023-12-06T22:59:59Z"`since Belgium is UTC+1 during winter time.

Reply:
indeed